Changelog.md

```diff
@@ -7,6 +7,7 @@ Compiler Features:
 
 
 Bugfixes:
+* CompilerStack: Fix crash that could occur when using the standard JSON interface for nondeployable contracts with ``outputSelection`` set to ``ir`` or ``irOptimized``.
```

Comment:
It didn't crash really, did it?

Reply:
It did report an error and nothing else beyond that. So not crash in the classical segfault sense, but more in the unhandled exception type. Reworded "crash" to "error" nonetheless.

<details>
Example:

```
{
    "errors": [
        {
            "component": "general",
            "formattedMessage": "Internal exception in StandardCompiler::compile: /solidity/libsolidity/interface/CompilerStack.cpp(821): Throw in function solidity::yul::YulStack solidity::frontend::CompilerStack::loadGeneratedIR(const string&) const\nDynamic exception type: boost::wrapexcept<solidity::langutil::InternalCompilerError>\nstd::exception::what: \n\nInvalid IR generated:\nError: Expected keyword \"object\".\n\n\n\n[solidity::util::tag_comment*] = \n\nInvalid IR generated:\nError: Expected keyword \"object\".\n\n\n\n",
            "message": "Internal exception in StandardCompiler::compile: /solidity/libsolidity/interface/CompilerStack.cpp(821): Throw in function solidity::yul::YulStack solidity::frontend::CompilerStack::loadGeneratedIR(const string&) const\nDynamic exception type: boost::wrapexcept<solidity::langutil::InternalCompilerError>\nstd::exception::what: \n\nInvalid IR generated:\nError: Expected keyword \"object\".\n\n\n\n[solidity::util::tag_comment*] = \n\nInvalid IR generated:\nError: Expected keyword \"object\".\n\n\n\n",
            "severity": "error",
            "type": "InternalCompilerError"
        }
    ]
}
```
</details>

Reply:
For future reference: we used "ICE" in a lot of cases in the Changelog, for Internal Compiler Error. But fine either way as is as well! I'd just usually have reserved "crash" for actual segfaults, but we have used it differently as well in the past.

Reply:
Ahh, right, ICE is much better:) cheers!

Reply:
Same. I usually say "internal compiler error" when it's a proper ICE. Crash when it's a segfault or random uncaught exception.